drud/ddev
# Allow PHPMyAdmin and Mailhog ports to be changed, fixes #1025

## The Problem/Issue/Bug:

OP #1025: It's long been asked by a few people to configure the PHPMyAdmin and Mailhog ports; this adds those to the config.yaml.

## How this PR Solves The Problem:

## Manual Testing Instructions:

use 
```
phpmyadmin_port: 10001
mailhog_port: 10002
```
in config.yaml. See that it works. Use `ddev describe` to see it and then visit the URL.

Also `ddev config --phpmyadmin-port=10001 --mailhog-port=10002`


## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:
